roles/tendrl-frontend/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+#
+# installation
+#
+
+- include: source.yml
+  when: install_from == "source"
+
+- include: binary.yml
+  when: install_from == "packages"
+
+#
+# post installation configuration
+#
```

Comment:
There is currently no post installation configuration. Something may be added later.